go.mod

```diff
@@ -20,6 +20,7 @@ require (
 	github.com/mgutz/ansi v0.0.0-20170206155736-9520e82c474b
 	github.com/pkg/errors v0.9.1
 	github.com/sclevine/spec v1.4.0
+	github.com/shurcooL/go v0.0.0-20180423040247-9e1955d9fb6e
```

Comment:
What is this used for?

Reply:
I think that showed up when I merged master in to this branch, I'm not using it for anything. It looks like it's being used in rebase/main.go

Reply:
@tomkennedy513 I ran into [this oddity](https://github.com/pivotal/kpack/pull/590#discussion_r552902337) too and @matthewmcnew fixed it with https://github.com/pivotal/kpack/pull/593. not approved/merged yet